### PR TITLE
OCLOMRS-1048: Back button goes back to dictionary instead of search results

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
# JIRA TICKET NAME:
[Pick concepts - Go back button goes back to dictionary instead of search results](https://issues.openmrs.org/browse/OCLOMRS-1048)

# Summary:
- Modify the backUrl for when you are importing

### CODE REVIEW

**Review by suruchee — APPROVED** [review]
Thank you @hadijahkyampeire  for the work, LGTM

File: src/apps/concepts/pages/ViewConceptPage.tsx [hunk 1]
```diff
@@ -67,6 +66,7 @@ const ViewConceptPage: React.FC<Props> = ({
             : "View concept"
         }
         backUrl={backUrl}
+
```
Comment (ibacher) [inline src/apps/concepts/pages/ViewConceptPage.tsx:69]:
```suggestion
```

File: src/apps/concepts/pages/ViewConceptPage.tsx [hunk 2]
```diff
@@ -54,8 +54,7 @@ const ViewConceptPage: React.FC<Props> = ({
     retrieveConcept(url);
   }, [url, retrieveConcept]);
 
-  const fromDictionary = linkedDictionary ?? dictionaryToAddTo;
-  const backUrl = `${fromDictionary ? fromDictionary : conceptSource}concepts/`
+  const backUrl = linkedDictionary ? `${linkedDictionary}concepts/` : dictionaryToAddTo ? undefined : `${conceptSource}concepts/`;
```
Comment (ibacher) [inline src/apps/concepts/pages/ViewConceptPage.tsx:57]:
```suggestion
  const backUrl = linkedDictionary
    ? `${linkedDictionary}concepts/`
    : dictionaryToAddTo
    ? undefined
    : `${conceptSource}concepts/`;
```

**Review by ibacher — APPROVED** [review]

**Comment by hadijahkyampeire** [comment]
Thanks @ibacher for the review.

**Pull request merged**